triage_bot/lib/discord.dart

```diff
@@ -0,0 +1,135 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
Update the copyrights?

Reply:
This file was started by copying from a file that had a 2019 copyright, so it would be wrong to change the date.